stream_alert/rule_processor/handler.py

```diff
@@ -16,20 +16,21 @@
 
 class StreamAlert(object):
     """Wrapper class for handling all StreamAlert classificaiton and processing"""
-    def __init__(self, context, return_alerts=False):
+    def __init__(self, context, send_alerts=True):
         """
         Args:
             context: An AWS context object which provides metadata on the currently
                 executing lambda function.
-            return_alerts: If the user wants to handle the sinking
-                of alerts to external endpoints, return a list of
-                generated alerts.
+            send_alerts: If the user wants to send the alerts using their own
```

Comment:
@mime-frame just so I'm clear, by 'rotate your logic' you mean you want the alert processor to be **disabled** by default??